StoreTestWPF.ViewModel/ViewModels/CakeViewModel.cs

```diff
@@ -0,0 +1,66 @@
+﻿using StoreTestWPF.DAL.Models;
+
+namespace StoreTestWPF.ViewModel.ViewModels
+{
+    public class CakeViewModel : ViewModelBase
+    {
+        private Cake cake;
+
+        public CakeViewModel(Cake cake)
+        {
+            this.cake = cake;
+        }
+
+        public Cake Cake { get { return this.cake; }}
```

Comment:
Lets use another notation approach:
public Cake Cake => this.cake;

Reply:
Please change in all places.

Reply:
Fixed